config.example.json

```diff
@@ -1 +1,4 @@
-{}
+{
+  "BASE_URL": "https://coronastatus.no",
+  "IMAGE_SUBFOLDER": "no"
```

Comment:
Maybe have a more general setting that we can use for multiple things, `LANGUAGE` for instance?

Reply:
Yup, sounds good!

Reply:
Changed now